tools/omero/omero_filter.xml

```diff
@@ -108,6 +108,12 @@ Tool to filter images IDs by filename, Key-Value Pairs and Tag value.
 For Key-Value Pairs search, two values are required (Value1 = Key, Value2 = Pair).
 IDs are a list of image IDs which can be fetched using the omero_get tool.
 
+**OMERO-suite Security Diclaimer:** To utilize the OMERO tools, the user must trust Galaxy instances.
+The configuration file, which contains your OMERO password and username, is stored in the job working directory.
+This directory only exists during the runtime of the job and should only be accessible by the system user that runs the job.
+However, please be aware that your username and password may be exposed to users with administrative rights.
+We are working on increasing the security of the OMERO suite
+
```

Comment:
Define a macro token in `macros.xml` (e.g. `@SECURITY_DISCLAIMER@` ) and reuse it in the tools.